website/docs/d/pi_volume_flash_copy_mappings_html.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 subcategory: "Power Systems"
 layout: "ibm"
-page_title: "IBM: pi_volume_flash_copy_mappings"
+page_title: "IBM: ibm_pi_volume_flash_copy_mappings"
```

Comment:
No need to change this line. Everywhere it starts with `pi`.
I guess this will change the datasource URL in the registry.

Reply:
updated